### ISSUE TITLE
No translation available if the app is initialized with other language than 'en'

### ISSUE THREAD

**Issue by akolybelnikov** [ngstack/core#2]
If I return a different language than 'en' in the `setupTranslateFactory`, no translation is available on the app initialization: (1) visual debugging shows 'en' instead; (2) strings are not translated by the pipe.

Setting the `activeLang` explicitly in the `setupTranslateFactory` like so:
    `service.activeLang = 'fr'` resolves the issue. 

Can it have something to do with the fact that the `data` variable is an empty object initially and is being returned by the async `use` earlier than the `await load` is finished, perhaps? If you add
     `.then(response => console.log(response))` to the `setupTranslateFactory`, there is the correct response.

If you want to quickly reproduce the issue, swap the 'en' for 'fr' in the [Stackblitz for Angular 6](https://stackblitz.com/edit/ngstack-translate-angular6?file=src%2Fapp%2Fapp.module.ts).

**Comment by DenisVuyka** [comment]
The `use` method is meant to manipulate and load data, for example `.use('en')` instructs to load `en.json`, however `.use('en', { "some": "value" })` bypasses http and assigns data directly (great for unit testing or programmatic merging).

The `activeLang` should be the primary way changing the locale. You don't need even calling `use` as service will load the necessary files itself.

Example:

```
export function setupTranslateFactory(service: TranslateService): Function {
  service.activeLang = 'fr';
  return () => Promise.resolve(true);
}
```

**Comment by akolybelnikov** [comment]
I tried it in a few ways, and it doesn't bring a consistent result: what I do, I'm setting the language dynamically based on user's preference which I get from the local storage. Using `Promise.resolve(true)` yields well with a language other than English. Once I set a safety check to use 'en' as a fallback, the translation file is not getting loaded. 
If I return like so:
`return () => service.use(user_language || 'en')`
all loads well.

**Comment by DenisVuyka** [comment]
Are there any steps to reproduce the issue?

**Comment by DenisVuyka** [comment]
@akolybelnikov I've just released new version [0.4.0](https://www.npmjs.com/package/@ngstack/translate). There's an improved `forRoot` method with params and no need setting up the factories.

Could you please try with something like:

```ts
TranslateModule.forRoot({
      activeLang: 'en'
})
```
